api/sheets/places_in.md

```diff
@@ -6,53 +6,33 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Retrieve places contained within another place
+# Retrieving Child Nodes
 
-Given a list of parent [`Place`](https://datacommons.org/browser/Place) [DCIDs](/glossary.html),
-(e.g. any [`State`](https://datacommons.org/browser/State), [`Country`](https://datacommons.org/browser/Country), etc.), return a list of child places
-contained within the specified DCIDs. Only returns children whose place type matches
-the request's `placeType` parameter.
+The `=DCPLACESIN(*dcids*, *placeType*) formula returns lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html), such as [State](https://datacommons.org/browser/State), [Country](https://datacommons.org/browser/Country), and so on. It only returns children with a place type that matches the <code>placeType</code> parameter.
 
-## General information about this endpoint
+> **Note**:
+> Be sure to follow the instructions for Installing and Enabling the Sheets Add-On before using this formula.
 
-**Formula**: `=DCPLACESIN(dcids, placeType)`
+## Formula {#formula}
 
-**Required arguments**:
+```
+=DCPLACESIN(dcids, placeType)
+```
 
-*   `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+## Required Arguments {#required-arguments}
 
-*   `placeType`: The type of the contained (child) `Place` nodes to filter by. For example, `City` and `County` are contained within `State`. For a
-    full list of available types, see [the Data Commons graph browser entry for `Place`](https://datacommons.org/browser/Place).
+* `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+* `placeType`: The type of the contained (child) `Place` nodes to filter by. For example,`City` and `County`are contained within `State`. For a full list of available types, see [the Data Commons graph browser entry for Place](https://datacommons.org/browser/Place).
 
+## Examples {#examples}
 
-## Assembling the information you will need to use this formula
+This section contains examples of using the `=DCPLACESIN(*dcids*, *placeType*)` formula to return lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html), such as [State](https://datacommons.org/browser/State), [Country](https://datacommons.org/browser/Country), and so.
 
-This endpoint requires the argument `dcids`. [DCIDs](/glossary.html) are unique node identifiers defined by Data Commons. Your query will need to specify the DCIDs for the parent places of interest.
+### Example 1: Retrieve a List of Counties in Delaware
```

Comment:
Please add descriptions for these examples like the GETNAME page.

Reply:
Bo said he would prefer creating videos for each example, although I do like the example descriptions in the GETNAME page. Please let me know what you decide.

Reply:
We should still have text descriptions, even if there are videos or images.

Reply:
I'll add them